provider/views.py

```diff
@@ -255,7 +327,36 @@ def error_response(self, request, error, **kwargs):
 
         return self.render_to_response(ctx, **kwargs)
 
+    def get_implicit_response(self, request, client):
+        """
+        Assumes validated and authorized request with a response_type that is
+        associated with an implicit flow.
+        """
+        data = self.get_data(request)
+
+        lookup_kwargs = {
+            "user": request.user,
+            "client": client,
+            "scope": scope.to_int(*data.get('scope', '').split())
+        }
+
+        if constants.SINGLE_ACCESS_TOKEN:
+            token = self.get_access_token(request, **lookup_kwargs)
+        else:
+            token = self.create_access_token(request, **lookup_kwargs)
+
+        response_data = self.access_token_response_data(token, data['response_type'])
+
+        basepath = data.get("redirect_uri")
+        if not basepath:
+            basepath = token.client.redirect_uri
+        if len(data.get('state', '')) > 0:
+            response_data['state'] = data.get('state')
+        path = "{0}#{1}".format(basepath, urlencode(response_data))
```

Comment:
No. This is for the browser-only flow. The `#` is used to prevent data leaking.

You can read more in the oauth spec here: https://tools.ietf.org/html/rfc6749#section-4.2.2